Candy.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <iostream>
+#include <string>
+using namespace std;
+
+enum Type {
+	Bar,
+	Button,
```

Comment:
why red Button

Reply:
Unfortunately, i wasn't able to resolve this problem.